subsys/net/lib/lwm2m/lwm2m_obj_server.c

```diff
@@ -78,6 +100,24 @@ static struct lwm2m_engine_obj_field fields[] = {
 	/* Mark Transport Binding is RO but BOOTSTRAP needs to write it */
 	OBJ_FIELD_DATA(SERVER_TRANSPORT_BINDING_ID, RW, STRING),
 	OBJ_FIELD_EXECUTE(SERVER_REG_UPDATE_TRIGGER_ID),
+	#if defined(CONFIG_LWM2M_SERVER_OBJECT_VERSION_1_1)
```

Comment:
The preprocessor directives shouldn't be indented.

Reply:
OK I will fix that.